google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineAssetProvider.java

```diff
@@ -44,4 +46,25 @@ public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Ar
 
     return null;
   }
+
+  @Nullable
+  @Override
+  public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Module module) {
+    VirtualFile[] contentRoots = ModuleRootManager.getInstance(module).getContentRoots();
+
+    for (VirtualFile contentRoot : contentRoots) {
+      // TODO clean this up - do we need something more robust?
+      VirtualFile descriptorFile
+          = contentRoot.findFileByRelativePath("web/WEB-INF/appengine-web.xml");
```

Comment:
Ideally, but I didn't see a good way exposed by the framework. Given the module content root, I could do a recursive directory search for it, but that's not ideal. I'll check again to make sure there's no helper for this or if its indexed somehow.